charts/sscs-cor-frontend/values.preview.template.yaml

```diff
@@ -1,23 +1,23 @@
-tags:
-  - idam-pr
-idam-pr:
-  redirect_uris:
-    SSCS COR:
-    - https://${SERVICE_FQDN}/sign-in # this is the application callback URL where IDAM will send back the authentication code. SERVICE_FQDN is injected by Jenkins.
 nodejs:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   environment:
-    SSCS_API_URL: "http://sscs-cor-backend-aat.service.core-compute-aat.internal"
-    IDAM_URL: "https://idam-web-public.aat.platform.hmcts.net"
-    IDAM_API_URL: "https://idam-api.aat.platform.hmcts.net"
-    TRIBUNALS_API_URL: "http://sscs-tribunals-api-aat.service.core-compute-aat.internal"
-    IDAM_CLIENT_SECRET: ${IDAM_CLIENT_SECRET}
-    S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
     REDIS_URL: redis://${SERVICE_NAME}-redis-master
     SESSION_SECRET:
     NODE_ENV: preview
```

Comment:
can probably bin this?

Reply:
Had some references in code, thought i better leave it there.